conflation/trace_fetching/mapillary.py

```diff
@@ -281,3 +285,100 @@ def make_trace_data_requests(
     [s.reverse() for s in sequences]
 
     return sequences
+
+
+def split_bbox(
```

Comment:
This function used to be in the common `util.py`, but basically we're saying each API provider should implement their own way of splitting the original given bbox.